src/app/pages/HomePage/HomePage.tsx

```diff
@@ -3,28 +3,16 @@
 import React, { useContext } from 'react';
 import { jsx } from '@emotion/react';
 import VisuallyHiddenText from '#app/components/VisuallyHiddenText';
-import {
-  VisualProminence,
-  VisualStyle,
-  CurationData,
-} from '../../models/types/curationData';
+import ATIAnalytics from '#app/components/ATIAnalytics';
+import { VisualProminence, VisualStyle } from '../../models/types/curationData';
 import Curation from '../../components/Curation';
 import { ServiceContext } from '../../contexts/ServiceContext';
 import styles from './index.styles';
 import MetadataContainer from '../../components/Metadata';
 import LinkedData from '../../components/LinkedData';
 import getItemList from '../../lib/seoUtils/getItemList';
 import ChartbeatAnalytics from '../../components/ChartbeatAnalytics';
-
-interface HomePageProps {
-  pageData: {
-    pageType: string;
-    id?: string;
-    title: string;
-    curations: CurationData[];
-    description: string;
-  };
-}
+import { HomePageProps } from '../../components/ATIAnalytics/types';
```

Comment:
I think we should leave the HomePageProps in this file

Reply:
Reverted!